### PR TITLE
Add source relative option for protoc-gen-validate

### PR DESCRIPTION
**Pull request by heliuchuan**
Adds a flag to enable the output path for protoc-gen-validate to be source relative.  Otherwise it will use the directory specified in option go_package.

### CODE REVIEW

File: all/entrypoint.sh [hunk 1]
```diff
@@ -136,6 +137,10 @@ while test $# -gt 0; do
             fi
             shift
             ;;
+	--validator-source-relative)
```
Comment (ido-namely) [inline all/entrypoint.sh:140]: Is there any reason for the validator files to be source relative while other generated code is not source relative, 
or can the `go-source-relative` flag be used to set the value for the validator?

**Review by ido-namely — COMMENTED** [review]
Hi @heliuchuan , thanks for your pr.
I left a comment/question.

File: all/entrypoint.sh (hunk 1 above)
Comment (heliuchuan) [reply all/entrypoint.sh:140]: Indeed, I did consider using that flag...however I didn't want to check in something that was not backwards compatible.  
For us I don't believe there would be a reason but I can't speak for everyone.

Comment (heliuchuan) [reply all/entrypoint.sh:140]: @ido-namely ping

Comment (ido-namely) [reply all/entrypoint.sh:140]: Hi @heliuchuan, sorry for the delay.
I'll try to test your changes tonight and if everything works well I'll merge this.

**Review by ido-namely — COMMENTED** [review]
Hi @heliuchuan ,
Sorry for the delay, I tested your changes and I would be happy to approve and cut 1.33_1 
if you don't mind changing [this](https://github.com/namely/docker-protoc/blob/4833be2f2269ad15ff57b86aa861291d93811f4d/variables.sh#L9) value to `1` first.

**Review by ido-namely — COMMENTED** [review]
Thanks for the contribution.

**Pull request merged**